docs/section5/Rule5-42.md

```diff
@@ -0,0 +1,35 @@
+# Envelope - Rule 5-42  
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** False    
+**Rule ID:** 5-42  
+ 
+**Rule Description:** Each linear thermal bridge and point thermal bridge as identified in Section 5.5.5 shall be modeled using either of the following techniques:  
+
+a. A separate model of the assembly within the energy simulation model.  
+b. Adjustment of the clear-field U-factor in accordance with Section A10.2.    
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Table G3.1 Section 5(a)-1 Proposed  
```

Comment:
This is technically an exception so recommend Table G3.1 Section 5a Exception #1.